### PR TITLE
Removed modeler.primitives from unit tests and examples as it is deprecated.

### PR DESCRIPTION
**Pull request by maxcapodi78**
…ecated and now any methods from primitives can be called directly from modeler

eg. modeler.primitives.create_box becomes modeler.create_box

### CODE REVIEW

File: _unittest/test_02_2D_modeler.py [hunk 1]
```diff
@@ -124,14 +124,14 @@ def test_create_regular_polygon(self):
 
     @pytest.mark.skipif(config["build_machine"] or is_ironpython, reason="Not running in ironpython")
     def test_plot(self):
-        self.aedtapp.modeler.primitives.create_regular_polygon([0, 0, 0], [0, 2, 0])
-        self.aedtapp.modeler.primitives.create_regular_polygon(
+        self.aedtapp.modeler.create_regular_polygon([0, 0, 0], [0, 2, 0])
```
Comment (Alberto-DM) [inline _unittest/test_02_2D_modeler.py:127]: There are some lines, like this one, without an assert. Any reason why there is no assert? 
They are notified by codecov.
Maybe we can exclude all unit tests from code coverage.

**Review by Alberto-DM — APPROVED** [review]
I made one comment. Other than that it looks good to me.

File: _unittest/test_02_2D_modeler.py (hunk 1 above)
Comment (maxcapodi78) [reply _unittest/test_02_2D_modeler.py:127]: because those one are needed to the test assertion which is a couple of lines below

**Review by MaxJPRey — APPROVED** [review]

**Comment by MaxJPRey** [comment]
LGTM. Well Done @maxcapodi78 .

**Pull request merged**